<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainUI</class>
 <widget class="QMainWindow" name="mainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PC-BSD Device Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pc-devicemanager.qrc">
    <normaloff>:/images/harddrive.png</normaloff>:/images/harddrive.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>WARNING: Everything changed here is permanent.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Make sure that you have your data backed up elsewhere first!</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_removable">
       <attribute name="icon">
        <iconset resource="pc-devicemanager.qrc">
         <normaloff>:/images/usb.png</normaloff>:/images/usb.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Removable Devices</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="combo_rd_devices">
            <property name="statusTip">
             <string>Currently attached devices (must be unmounted)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tool_rd_refresh">
            <property name="statusTip">
             <string>Refresh the list of available devices</string>
            </property>
            <property name="text">
             <string>Refresh</string>
            </property>
            <property name="icon">
             <iconset resource="pc-devicemanager.qrc">
              <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QGroupBox" name="group_rd_current">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Current Configuration</string>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::ExpandingFieldsGrow</enum>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Label:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_rd_label">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Size:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_rd_size">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Partitions:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="label_rd_partitions">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="group_rd_changes">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Changes To Make</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QComboBox" name="combo_rd_changelist">
               <property name="statusTip">
                <string>Possible changes for the selected device</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QStackedWidget" name="stack_rd_changepages">
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="page_reformat">
                <layout class="QFormLayout" name="formLayout_3">
                 <property name="fieldGrowthPolicy">
                  <enum>QFormLayout::ExpandingFieldsGrow</enum>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Device Label:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="line_rd_format_label">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="statusTip">
                    <string>The new name by which to recognize the device.</string>
                   </property>
                   <property name="inputMethodHints">
                    <set>Qt::ImhDigitsOnly|Qt::ImhLowercaseOnly|Qt::ImhPreferUppercase|Qt::ImhUppercaseOnly</set>
                   </property>
                   <property name="maxLength">
                    <number>11</number>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>Filesystem:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QComboBox" name="combo_rd_format_fs">
                   <property name="statusTip">
                    <string>The new filesystem to put on the device</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <spacer name="verticalSpacer">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="3" column="1">
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <spacer name="horizontalSpacer">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>10</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QToolButton" name="tool_rd_reformatdevice">
                     <property name="statusTip">
                      <string>Start the process of formatting the device</string>
                     </property>
                     <property name="text">
                      <string>Format Device</string>
                     </property>
                     <property name="icon">
                      <iconset resource="pc-devicemanager.qrc">
                       <normaloff>:/images/checkmark.png</normaloff>:/images/checkmark.png</iconset>
                     </property>
                     <property name="toolButtonStyle">
                      <enum>Qt::ToolButtonTextBesideIcon</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="page_2"/>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>552</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="pc-devicemanager.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="pc-devicemanager.qrc"/>
 </resources>
 <connections/>
</ui>
